microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -665,6 +670,16 @@ To turn off speech translated to text, you may deny microphone access from any s
 
 To help resolve URL typos in the address bar that result in a website error, Microsoft Edge may recommend a corrected URL.  When a website navigation error occurs, Microsoft Edge sends the domain of the web address to the Microsoft service to suggest a corrected URL.  Microsoft Edge does not include identifiers or tokens with the domain.  If the service finds a suggestion, it returns the suggested URL.  Microsoft stores the incorrect domain, and suggested domain, to help improve the service.  To help you go to the correct sites, the feature is turned on by default.  To turn off the feature, go to `edge://settings/privacy` and under the **Services** turn off the **Suggest similar sites when a website can't be found** setting.
 
+<!-- ====================================================================== -->
+## Support nonprofits
+Microsoft Edge allows you to support nonprofit organizations using Microsoft Rewards points or cash while browsing. When you are signed in to Microsoft Edge with your Microsoft account and navigate to a nonprofit website, the support nonprofits icon will appear on the right-side of your address bar. You can then click this icon to donate to the nonprofit organization.
+
+![Image of support nonprofits icon in address bar.](./media/support-nonprofit.png)
```

Comment:
```suggestion
![The 'Support nonprofits' icon in the address bar](./media/support-nonprofit.png)
```
The default / common for .pngs in this repo is a screenshot image, not a diagram, so no need to specify - 
wiki > Edge Dev Style Guide, probably covers this.